tests/acceptance-tests/src/acceptance/http_closed_test.go

```diff
@@ -31,13 +31,13 @@ var _ = Describe("Http is closed", func() {
 
 		uri := appName + "." + config.AppsDomain + ":80"
 		_, err := net.DialTimeout("tcp", uri, CONNECTION_TIMEOUT)
-		Expect(err.(net.Error).Timeout()).To(BeTrue())
+		Expect(err).To(HaveOccurred())
```

Comment:
I'd prefer this to still check for an exact error. Otherwise there's a risk that it could return a false-positive result e.g. if it was unable to resolve the hostname.

I think the problem with the test panicing on a successful connection could be solved by an additional assertion of:
```golang
Expect(err).To(BeAssignableToTypeOf(net.Error))
```

The problem of it returning a TCP RST when run from outside the environment is more tricky. I guess we'd need to check whether it's either a timeout or equivalent to `syscall.ECONNRESET`.